src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -29,6 +29,7 @@
 use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
 use PrestaShop\PrestaShop\Core\ConfigurationInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Validate;
```

Comment:
ping @matks shouldn't we inject the Validate adapter and add the missing `isEmail` method?

Reply:
Yes, like @PululuK did in https://github.com/PrestaShop/PrestaShop/pull/24622/